SpatialGDK/Documentation/content/get-started/porting-unreal-project-to-gdk.md

```diff
@@ -89,26 +107,31 @@ Before porting your project:
     ```
 
 ### 3. Build your project
-Set up your Unreal game project to work with the GDK for Unreal fork of the Unreal Engine, which you cloned and installed in the [Before you start](#before-you-start) section. To do this:
+Set up your Unreal project to work with the GDK for Unreal fork of the Unreal Engine, which you cloned and installed in the [Before you start](#before-you-start) section. To do this:
 
-1. In File Explorer, navigate to `<ProjectRoot>\<GameRoot>`.
-1. Right-click your `.uproject` file and select **Switch Unreal Engine version**.
+1. In **File Explorer**, navigate to `<ProjectRoot>\<GameRoot>`.
+1. Right-click your `<YourProject>.uproject` file and select **Switch Unreal Engine version**.
 1. Select the path to the Unreal Engine fork you cloned earlier.
-1. In File Explorer, right-click your `.uproject` file and select **Generate Visual Studio project files**.
-1. Open the generated solution file in Visual Studio and in the Development Editor build configuration, compile and run the project.
+1. In **File Explorer**, right-click `<YourProject>`.uproject and select **Generate Visual Studio Project files**.
+1. In the same directory, double-click `<YourProject>`.sln to open it with Visual Studio.
```

Comment:
This `sln` file is generated by the previous step. Is it worth referencing that fact?

Reply:
```suggestion
1. In **File Explorer**, right-click `<YourProject>`.uproject and select **Generate Visual Studio Project files**. This automatically generates a Visual Studio solution file for your project called `<YourProject.sln>`
1. In the same directory, double-click `<YourProject>`.sln to open it with Visual Studio.
```

Does this work?

Reply:
Looks good!